docs/test/textbox.html

```diff
@@ -5,25 +5,27 @@
 			  visibility: hidden;
 			}
 			input.notdisplayed {
-  				display: none;
+				display: none;
 			}
 		</style>
 	</head>
 	<body>
 		<form action="/action_page.php">
-		  <input type="checkbox" id="vehicle1" name="vehicle1" value="Bike">
-		  <label for="vehicle1"> I have a bike</label><br>
-		  <input type="checkbox" id="vehicle2" name="vehicle2" value="Car">
-		  <label for="vehicle2"> I have a car</label><br>
-		  <input type="checkbox" id="vehicle3" name="vehicle3" value="Boat" checked>
-		  <label for="vehicle3"> I have a boat</label><br><br>
-		  <label for="fname">First name:</label>
-		  <input type="text" id="fname" name="fname" disabled><br><br>
-		  <label for="lname">Last name:</label>
-		  <input type="text" id="lname" name="lname"><br><br>
-		  <input type="submit" value="Submit">
-		  <input type="text" id="hiddenbox" class="hidden">
-		  <input type="text" id="notdisplayedbox" class="notdisplayed">
+			<input type="checkbox" id="vehicle1" name="vehicle1" value="Bike">
```

Comment:
Is the tabbing off or is this just a visual thing here?

Reply:
looks like it's been changed to be more consistent with the rest of the document. Don't think it's a problem so I'm fine with it.

Reply:
Yeah it wasn't quite tabbed the right way before. Same with the page object which used non standard tabbing before.